swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+from uuid import uuid4
+
+CONDITION_DESCRIPTION = { 0: "heavily used", 
+                        1: "used", 
+                        2: "fair", 
+                        3: "good", 
+                        4: "like new", 
+                        5: "mint"}
+
+
+#wave 2
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        self.id = uuid4().int if id is None else id
+        self.condition = condition
+
+
+    def get_category(self):
+        return "Item" #returnig class as a string
```

Comment:
A dev reading line 19 would understand that `return "Item"` is returning a string that represents the class name. I super would prefer that these simple comments are removed before submitting your code for review. 

```suggestion
        return "Item"
```

The comment doesn't necessarily add clarity and every place we add a comment, we increase the amount of a codebase that needs to be maintained. 

In industry each team will have their preference about what kind of comments and how many are allowed, but typically you will see them used very, very sparingly.

It would be great practice to wean yourself off comments like this as you continue developing as a software engineer.